nilearn/surface/surface.py

```diff
@@ -895,6 +897,65 @@ def load_surf_mesh(surf_mesh):
     return mesh
 
 
+def load_surface(surface):
+    """Loads a surface.
+
+    Parameters
+    ----------
+    surface : List of numpy.ndarray or Surface
+        Surface to be loaded. This can be passed as:
+
+        - a list of two files (valid formats are .gii
+        .gii.gz or Freesurfer specific files such as
+        .orig, .pial, .sphere, .white, .inflated) containing:
+            - a surface mesh geometry
+            - surface data
+        - three Numpy arrays organized in a list with coordinates,
+        faces, and data in this specific order
+        - a length 2 tuple or a namedtuple with the fields "mesh" and "data"
```

Comment:
there is no such thing as "a namedtuple", you can say any object with attributes "mesh" and "data", or a Surface